.github/workflows/build.yml

```diff
@@ -0,0 +1,25 @@
+name: CI
+
+on: 
+  push:
+
```

Comment:
Just checking - will this actually trigger on pushes?

Reply:
Seeing as the checks are running on this PR, yes!